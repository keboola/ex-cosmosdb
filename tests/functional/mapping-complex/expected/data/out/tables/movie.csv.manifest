{
    "columns": [
        "title-id",
        "release_date",
        "missing",
        "directors"
    ],
    "primary_key": [
        "title-id"
    ],
    "incremental": false
}
