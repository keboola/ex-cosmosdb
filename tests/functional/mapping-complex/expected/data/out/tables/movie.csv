"Rush","2013-09-02T00:00:00Z","","[""Ron Howard""]"
"Prisoners","2013-08-30T00:00:00Z","","[""Denis Villeneuve""]"
"Insidious: Chapter 2","2013-09-13T00:00:00Z","","[""James Wan""]"
